Alignment/OfflineValidation/plugins/PrimaryVertexValidation.cc

```diff
@@ -283,10 +283,11 @@ void PrimaryVertexValidation::analyze(const edm::Event& iEvent, const edm::Event
   //edm::Handle<VertexCollection> vertices;
   edm::Handle<std::vector<Vertex>> vertices;
 
-  try {
-    vertices = iEvent.getHandle(theVertexCollectionToken_);
-  } catch (cms::Exception& er) {
-    LogTrace("PrimaryVertexValidation") << "caught std::exception " << er.what() << std::endl;
+  vertices = iEvent.getHandle(theVertexCollectionToken_);
+  if (!vertices.isValid()) {
+    edm::LogError("PrimaryVertexValidation") << "Vertex collection handle is not valid. Aborting!";
+
```

Comment:
Since you are cleaning, this line can also get removed:
```suggestion
```

Reply:
I don't know how to do it from web UI, so the commits are squashed from my local git CLI.